docs/YOSEMITE.md

```diff
@@ -34,6 +34,8 @@ Each subclass of `Store` registers a processor with the `Dispatcher`. To registe
 
 That set of actions is declared in an implementation of the `Action` protocol. We will look at `Action` in details in the next section, but for now,  it is important to note that there is a one to one relationship between each subclass of `Store` and a corresponding implementation of the `Action` protocol. (i.e. for `AccountStore` we would also have a `AccountAction`).
 
+Stores can also expose `Combine` publishers for read-only access to data that's not persisted in Core Data. For accessing managed objects, the preferred method is to rely on `EntityListener` or `ResultsControllers`. Any publishers exposed by a store can be accessed through the `publishers(keyPath:)` method of the `StoresManagerState`.
```

Comment:
Writing this, I'm now wondering if it makes sense to also add a `publisher(keyPath:)` method to the Store Manager so the UI layer can request a key path directly.

Right now, exposing `connectedReaders` as a shorthand reads more nicely, but if this become useful in more scenarios, then the store manager might start to have too many properties. 🤔 

```swift
// With publisher
ServiceLocator.stores.publisher(keyPath: \CardPresentPaymentStore.connectedReaders)
// With shorthand
ServiceLocator.stores.connectedCardReaders
```